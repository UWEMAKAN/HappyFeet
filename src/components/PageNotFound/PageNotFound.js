import React from 'react';
import './PageNotFound.css';

const PageNotFound = () => (
  <div className="PageNotFound">
    404 ERROR
    <br />
    <br />
    PAGE NOT FOUND
  </div>
);

export default PageNotFound;
