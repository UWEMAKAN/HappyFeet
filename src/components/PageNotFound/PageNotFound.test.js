import React from 'react';
import { shallow } from 'enzyme';
import PageNotFound from './PageNotFound';

test('should render PageNotFound', () => {
  const wrapper = shallow(<PageNotFound />);
  expect.assertions(1);
  expect(wrapper).toMatchSnapshot();
});
