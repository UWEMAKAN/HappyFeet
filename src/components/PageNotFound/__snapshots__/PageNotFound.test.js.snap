// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PageNotFound 1`] = `ShallowWrapper {}`;
